apps/frontend/src/pages/people/[username].tsx

```diff
@@ -182,10 +183,6 @@ const CommentModal: React.FC<CommentModalProps> = ({
 
         <div className="mt-4 grid grid-cols-1 gap-4">
           <AppButton
-            disabled={
-              (selectedEmoji ?? "") === (previousEmoji ?? "") &&
-              privateNote === (previousNote ?? "")
-            }
```

Comment:
I've gotten a ton of complaints from people about the connection comment save button being disabled until the emoji / note is added.